tool-populate/api/Roles.js

```diff
@@ -23,13 +24,12 @@ function loadRoles(options, callback) {
 }
 
 function createRole(roleToCreate) {
-  var roleACL = new Parse.ACL();
-  roleACL.setPublicReadAccess(true);
-
+  var acl = new Parse.ACL()
+  acl.setPublicReadAccess(false)
```

Comment:
I think a curious user could attach the 'user' or 'admin' role to their submission but not create/update/delete them.  If we set this to public read to false, then client queries will need to useMasterKey.